src/components/dashboard/ClaimButton.js

```diff
@@ -12,35 +12,43 @@ import { weiToGd } from '../../lib/wallet/utils'
 import { getDesignRelativeHeight, getDesignRelativeWidth } from '../../lib/utils/sizes'
 import { isSmallDevice } from '../../lib/utils/mobileSizeDetect'
 
-const buttonLabelFontSize = isSmallDevice ? 28 : 34
-const timerFontSize = isSmallDevice ? 24 : 30
-
 const ButtonAmountToClaim = ({ showLabelOnly = false, entitlement, isCitizen, styles, isInQueue }) => (
-  <View style={styles.textBtn}>
+  <View>
     {showLabelOnly ? (
       isInQueue ? (
         <Text
           style={{ letterSpacing: 0.28 }}
           color="white"
           fontFamily="Roboto Slab"
           fontWeight="bold"
-          fontSize={buttonLabelFontSize}
+          fontSize={28}
           textAlign="center"
         >
           In Queue
         </Text>
       ) : (
-        <Text
-          style={{ letterSpacing: 0.28 }}
-          color="white"
-          fontFamily="Roboto Slab"
-          fontWeight="bold"
-          fontSize={buttonLabelFontSize}
-          textAlign="center"
-        >
-          CLAIM <br />
-          NOW
-        </Text>
+        <>
+          <Text
+            style={{ letterSpacing: 0.28 }}
+            color="white"
+            fontFamily="Roboto Slab"
+            fontWeight="bold"
+            fontSize={28}
+            textAlign="center"
+          >
+            CLAIM
+          </Text>
+          <Text
+            style={{ letterSpacing: 0.28 }}
+            color="white"
+            fontFamily="Roboto Slab"
+            fontWeight="bold"
+            fontSize={28}
+            textAlign="center"
+          >
+            NOW
+          </Text>
```

Comment:
why not merge these two Text components into one Text component with the text "CLAIM\nNOW"? since they both have exactly the same styling